### ISSUE TITLE
No gamepad input while in game mode (gamescope) when adding games to Steam via Lutris

### ISSUE THREAD

**Issue by RodoMa92** [ublue-os/bazzite#1500]
### Describe the bug

I can't get any gamepad input while using the provided Lutris installed software while starting games on game mode. If I enable gamescope from the lutris settings the game will completely fail to start with a weird non gamescope layer error. Without it enabled the game start fine, but with no gamepad input available. On desktop mode both (non gamescope/gamescope enabled in lutris) work 100% fine in my testing.

### What did you expect to happen?

Gamepad inputs working in game mode for games installed via Lutris

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:stable
                   Digest: sha256:6ce12036986eb6b37658dac3d335ce826106cf46a628516be96bf966bc8f6cbf
                  Version: v3.6-40.20240809.0 (2024-08-09T21:14:34Z)
          LayeredPackages: android-tools aspell-it bchunk bluez-tools
                           cdemu-client cdemu-daemon fluidsynth gh kde-cdemu-manager-kf6
                           patch powerline-fonts qt5pas rasdaemon rmlint smb4k sqlite-devel
                           strace tldr valgrind xlsclients yt-dlp zsh
                Initramfs: '"-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf"' 

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:stable
                   Digest: sha256:fa9d7c77f4be2d451aa62cc232649afd087a2ea0f539df891ef4bb5cb6ac488a
                  Version: 40.20240807.0 (2024-08-07T07:32:24Z)
          LayeredPackages: android-tools aspell-it bchunk bluez-tools
                           cdemu-client cdemu-daemon fluidsynth gh kde-cdemu-manager-kf6
                           patch powerline-fonts qt5pas rasdaemon rmlint
                           setroubleshoot-server smb4k sqlite-devel strace tldr valgrind
                           xlsclients yt-dlp zsh
                Initramfs: '"-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf"'
```

### Hardware

Jupiter

### Extra information or context

From what I've seen [here](https://github.com/ValveSoftware/gamescope/issues/1180#issuecomment-2106862408), xwayland + libei is required to get this to work. Is this correctly enabled in the bazzite builds of gamescope?

**Comment by fsworld009** [comment]
Observed the same issue on my Bazzite installation on GPD Win 4 as well.

I'm currently using Flatpak Lutris as workaround. With the exact same game installation step (manually configured Wine game), the Steam input works when launching games in Flatpak Lutris.

**Comment by RodoMa92** [comment]
Yep, can confirm that switching to flatpak as a workaround makes it work, since it's using flatpak gamescope instead of the built in one. However we lose additional fixes from bazzite, so this is not the ideal path to make it work.

**Comment by matte-schwartz** [comment]
> From what I've seen [here](https://github.com/ValveSoftware/gamescope/issues/1180#issuecomment-2106862408), xwayland + libei is required to get this to work. Is this correctly enabled in the bazzite builds of gamescope?

You would have no cursor in gamescope-session whatsoever if libei/input-emulation was disabled in the gamescope build.

**Comment by fsworld009** [comment]
libei seems to be enabled according to https://github.com/ublue-os/bazzite/blob/0f35915/spec_files/gamescope/gamescope.spec#L35 ?

Also, when I was testing this, I didn't enable any gamescope option in both Lutris and I don't have flatpak gamescope installe, so I'm not sure if the issue is related to gamescope.

**Comment by RodoMa92** [comment]
> libei seems to be enabled according to https://github.com/ublue-os/bazzite/blob/0f35915/spec_files/gamescope/gamescope.spec#L35 ?
> 
> Also, when I was testing this, I didn't enable any gamescope option in both Lutris and I don't have flatpak gamescope installe, so I'm not sure if the issue is related to gamescope.

When you run games in gamemode it will run them in gamescope by default, so since I doubt that there are other variables at play I've assumed that the embedded gamescope build is at fault here. Might be wrong though.

**Comment by dragaoazuljr** [comment]
I have the same issue but using Heroic Game Laucher, when starting the game using gamescope with the `-e` flag e.g `gamescope -f -W 1920 -H 1080 -e -- heroic --no-gui --no-sandbox "heroic://launch/gog/1495134320"`. Removing the `-e` flag the input works on the game, but steam overlay don't. What I found weird is that with the -e flag, games from steam works fine (Input, Overlay).

The same issue happens on another pc running a arch-based distro. So I don't think the problem is related to Bazzite.

**Comment by Zeglius** [comment]
> I have the same issue but using Heroic Game Laucher, when starting the game using gamescope with the `-e` flag e.g `gamescope -f -W 1920 -H 1080 -e -- heroic --no-gui --no-sandbox "heroic://launch/gog/1495134320"`. Removing the `-e` flag the input works on the game, but steam overlay don't. What I found weird is that with the -e flag, games from steam works fine (Input, Overlay).
> 
> The same issue happens on another pc running a arch-based distro. So I don't think the problem is related to Bazzite.

wouldn't be that weird, given the long version of `-e` is `--steam`. 
Chances are there is some logic under that flag to improve compatibility with the steam client and games launched from there, but breaks whenever it is other than steam launching the game

**Comment by dragaoazuljr** [comment]
Both use proton, shouldn't be that different. I bet is some environment variable or some flag we are missing.
